spec.md

```diff
@@ -1617,6 +1634,50 @@ The CO MUST implement the specified error recovery behavior when it encounters t
 |-----------|-----------|-------------|-------------------|
 | Volume does not exist | 5 NOT_FOUND | Indicates that a volume corresponding to the specified `volume_id` does not exist. | Caller MUST verify that the `volume_id` is correct and that the volume is accessible and has not been deleted before retrying with exponential back off. |
 
+
+#### `ModifyVolume`
+
+A Controller plugin MUST implement this RPC call if plugin has MODIFY_VOLUME controller capability. This RPC allows the CO to change mutable key attributes of a volume. 
+
+This operation MUST be idempotent. The new mutable parameters in ModifyVolume can be different from the existing mutable parameters. 
+
+
+```protobuf
+message ModifyVolumeRequest {
+  option (alpha_message) = true;
+
+  // Contains identity information for the existing volume.
+  // This field is REQUIRED.
+  string volume_id = 1;
+
+  // Secrets required by plugin to complete modify volume request.
+  // This field is OPTIONAL. Refer to the `Secrets Requirements`
+  // section on how to use this field.
+  map<string, string> secrets = 2 [(csi_secret) = true];
+
+  // Plugins specific parameters to apply, passed in as opaque key-value
+  // pairs. This field is OPTIONAL. The Plugin is responsible for
+  // parsing and validating these parameters. COs will treat these
+  // as opaque. COs MUST specify the intended value of every mutable
+  // parameter. Absent values that were previously set MUST be
+  // interpretted as clearing the value.
```

Comment:
"clearing"

.. meaning that the absent mutable parameter is considered *deleted* (vs set to empty string)?

Reply:
it will be considered as deleted.

Reply:
Suggestion: Absent keys that were previously assigned a value MUST be interpreted as a deletion of the key from the set of mutable parameters.